packages/relay/src/lib/services/cacheService/cacheService.ts

```diff
@@ -184,7 +187,7 @@ export class CacheService {
    * @param {string} [requestIdPrefix] - The optional request ID prefix.
    * @returns {Promise<any>} A Promise that resolves with the cached value or null if not found.
    */
-  public get(key: string, callingMethod: string, requestIdPrefix?: string): any {
+  private getFromInternalCache(key: string, callingMethod: string, requestIdPrefix?: string): any {
     this.cacheMethodsCounter.labels(callingMethod, CacheService.cacheTypes.LRU, CacheService.methods.GET).inc(1);
 
     return this.internalCache.get(key, callingMethod, requestIdPrefix);
```

Comment:
Shouldn't we also make this method `async`, since it's using the `async get()` method from `this.internalCache`?

```typescript
private async getFromInternalCache(key: string, callingMethod: string, requestIdPrefix?: string): Promise<any> {
  return this.internalCache.get(key, callingMethod, requestIdPrefix);
}
```

Reply:
Yes, i fixed it, thanks